frontend-react/src/styles/_uswds_settings.scss

```diff
@@ -10,8 +10,8 @@ $rs-width: "widescreen";
     * these paths are from where the output css lives, in this case
     * /frontend-react/src/content/generated/global.out.css
     */
-    $theme-image-path: "~@uswds/uswds/img",
-    $theme-font-path: "~@uswds/uswds/fonts",
+        $theme-image-path: "@uswds/uswds/img",
```

Comment:
[non-blocking] looks like a tiny tab issue here.

Reply:
Unfortunately my local prettier keeps doing that based off the current rules.